### PR TITLE
Update translations for master

### PR DESCRIPTION
**Pull request by abbra**
This pull request updates translations from the Zanata master branch and also updates Contributing.txt list according to the commits since last December.

### CODE REVIEW

File: Contributors.txt [hunk 1]
```diff
@@ -4,140 +4,153 @@ The following people have contributed to the FreeIPA project.
...
-	Petr Čech
-	Xiao-Long Chen
-	Jan Cholasta
-	Yuri Chornoivan
+	Alexander Koksharov
+	Alexander Scheel
+	Alexey Slaykovsky
+	Amit Kumar
+	amitkuma
+	amitkumar50
```
Comment (tiran) [inline Contributors.txt]: Amit is multiple times in the file

File: Contributors.txt [hunk 2]
```diff
@@ -4,140 +4,153 @@ The following people have contributed to the FreeIPA project.
...
-	James Groffen
-	Oliver Gutierrez
-	Ondřej Hamada
-	Robbie Harwood
-	Nick Hatch
-	Christian Heimes
-	Jakub Hrozek
+	felipe
+	Felipe Barreto
+	Felipe Volpone
```
Comment (tiran) [inline Contributors.txt]: same with Felipe

**Comment by abbra** [comment]
@tiran I've updated `.mailmap` to include the other variants and regenerated Contributing.txt. I'll push updated `freeipa-tools` later today as I had to change the code in `update-contributors.py` to work with Python3.

**Comment by tiran** [comment]
master:

* 6c361d74c5344d34b4924434eb13a202f1a5fed8 Update translations from Zanata
* e05008d10484858b4c482953e30df5194aeee7ed Update mailmap
* 993fd4d7fc6206a5412e7fd1a22897a1244fb44d Update list of contributors and sort them alphabetically

**Pull request closed without merging**